### ISSUE TITLE
React does not recognize the `aspectRatio` prop on a DOM element

### ISSUE THREAD

**Issue by mikoSTAR** [yosbelms/react-progressive-loader#5]
Hi I got following error once used with React "16.4.1".
Is that something that you noticed as well? 

```
Warning: React does not recognize the `aspectRatio` prop on a DOM element. 
If you intentionally want it to appear in the DOM as a custom attribute,
 spell it as lowercase `aspectratio` instead. 
If you accidentally passed it from a parent component, 
remove it from the DOM element.
    in div (created by Bare)
    in Bare (created by Img)
```

**Comment by yosbelms** [comment]
Can you provide some code to reproduce this issue?

**Comment by mikoSTAR** [comment]
It happens once you try to use `aspectRatio` prop.
For example:
`<Img
  src='image.jpeg'
  placeholderSrc='image-placeholder.jpeg'
  aspectRatio='1'
/>`

**Comment by yosbelms** [comment]
fixed in 0.2.2
